modules/ROOT/pages/dw-dataformat-types.adoc

```diff
@@ -1,53 +1,51 @@
-== DataFormat Types (dw::extension::DataFormat)
+= DataFormat Types (dw::extension::DataFormat)
 
-[%headers, cols="1,3,6"]
 |===
 | Type | Definition | Description
 
 | DataFormat
-| `type DataFormat = { binaryFormat?: Boolean, defaultCharset?: String,
-fileExtensions?: Array<String&#62;,
-acceptedMimeTypes: Array<MimeType&#62;,
-reader: &#40;content: Binary,
-charset: String,
-settings: ReaderSettings&#41; &#45;&#62; Any,
-writer: &#40;value: Any,
-settings: WriterSettings&#41; &#45;&#62; Binary }`
-a|
-
-* `binaryFormat?: Boolean`: True if this is data format is represented in a
-binary representation instead of text. If not present, false.
-* `defaultCharset?: String`: The default charset of this format, if any.
-* `fileExtensions?: Array<String&#62;`: Returns the list of file extensions
-with the . &#40;".json", ".xml", etc...&#41; that should be assigned to this
-Data Format
-* `acceptedMimeTypes: Array<MimeType&#62;` The list of MimeTypes that are
-accepted.
-* `reader: (content: Binary, charset: String, settings: ReaderSettings) -> Any`:
-This function will be in charge of reading the raw content and transform it
-into the DataWeave canonical model.
-* `writer: (value:Any, settings:WriterSettings) -> Binary`: This function will
-be in charge of writing the DataWeave canonical model into Binary content.
+| `type DataFormat = { /**
+* True if this is data format is represented in a binary representation instead of text, if not present is false
```

Comment:
Need to fix the formatting of table in this one manually: modules/ROOT/pages/dw-dataformat-types.adoc